packages/backend/src/core/TransactionHistory.ts

```diff
@@ -50,6 +50,26 @@ export class TransactionHistory {
     }
   }
 
+  getNonRevertableTransactionHistory() {
```

Comment:
What is "NonRevertable" Transaction? All transactions can be reverted, including freeze and escape. We should also show them on user page if they are reverted.